pages/nft-book-store/new.vue

```diff
@@ -106,14 +109,19 @@ const error = ref('')
 const isLoading = ref(false)
 const connectStatus = ref<any>({})
 
+const mdEditorPlaceholder = ref({
+  en: 'Product description in English...',
+  zh: '產品中文描述...'
+})
+
 const classIdInput = ref(route.query.class_id as string || '')
 const prices = ref<any[]>([{
   price: 0,
   stock: Number(route.query.count as string || 0),
   nameEn: 'Standard Edition',
   nameZh: '標準版',
-  descriptionEn: 'Content of standard edition',
-  descriptionZh: '標準版內容'
+  descriptionEn: '',
+  descriptionZh: ''
```

Comment:
Why change to empty string

Reply:
I've changed the original description to be displayed as a placeholder in the textarea. 
The description itself doesn't have a default value because I think that the description of each edition should not be the same.